Modules/System Tests/Base64 Convert/src/Base64ConvertTest.Codeunit.al

```diff
@@ -48,6 +52,20 @@ codeunit 135041 "Base64 Convert Test"
         Assert.AreEqual(Base64SampleTextWithLineBreaks, ConvertedText, ConvertionToBase64Err);
     end;
 
+    [Test]
```

Comment:
Additional tests with Stream in/out. See other comments first

Reply:
There's no change to stream handling, textencoding with stream is already support by you guys :) (but not tested)